tools/free_energy/alchemical_run/gmx_fep.xml

```diff
@@ -0,0 +1,518 @@
+
+This will generate input .MDP files and run all the steps for all the FEP windows iteratively.
+
+
+_____
+
+
+.. class:: infomark 
+
+**Rules of Thumb for Intermediate States**  
+
+    - These rules are not the end-all set and you should be familiar with why each one is suggested before just accepting them.  
+    - Bonded terms can be modified/turned off linearly. This includes angle or bond force constants as well as unconstrained bond distances. 
+    - Constrained bonds should not change length. There are free energy changes that cannot be ignored affiliated with this action. 
+    - Maximize similarity between states by removing/decoupling as few atoms as possible. 
+    - Do not open and close rings. This supersedes the previous rule. 
+    - Statistical uncertainty between any neighboring states should be equal. Rather challenging to do, but it has been proven to have the lowest variance path if you can pull it off. 
+    - Deleting or adding atoms should always be done with a soft core potential. 
+    - Changes in parameters can be done linearly. 
+    - All charge on atoms must be turned off prior to atomic repulsion. Otherwise you can get an infinite attractive potential and crash your simulation.  
+    - Similarly for only changes in terms, it's generally more efficient to change electrostatic terms separate from Lennard-Jones terms. 
+    - More states is better than fewer. Variance shrinks rapidly with number of states. You want the difference between intermediaries to be between 2-3 kBT 
+
+Obviously you will be limited on CPU power. Fewer states also leads to more samples begin required from each state, so take this into account when deciding number of states as well.
+However, for MBAR and TI, it can be shown that spreading samples across multiple states does not significantly affect the uncertainty, since for TI, each state contributes less to 
+the total uncertainty, and in MBAR, data contributes to the statistical precision of states with similar values of lambda. 
+
+Shape of the variance does not significantly change with number of atoms, only magnitude. More intermediates will still be required for a large number of atoms to reduce statistical noise.  
+Charge should be maintained across all λ 
+
+Simply having charged molecules is fine, but the net of the system should remain constant. If you must change the net charge, there are complicated ways to do so.
+
+Short prototype simulations are recommended. Even as short as 100 ps, the prototypes can provide rough magnitude of variance estimates, 
+although will likely under-predict the free energy as many configurations remain unsampled. 
+
+
+_____
+
+
+.. class:: infomark
+
+**Input**
+
+       - .GRO input 
+       - .TOP input
+
+_____
+
+       
+.. class:: infomark
+
+**Output**
+
+       - TI/FEP data
+       - TI/FEP trajectory
+       - Report
+
+    ]]></help>
+    <citations>
+      <citation type="doi">10.1016/j.softx.2015.06.001</citation>
```

Comment:
I would cite the alchemistry.org website too

Reply:
I think this is a must, given that much of the help text is taken directly from the website